src/wallet/wallet.factory.js

```diff
@@ -9,16 +9,6 @@
         'WalletStatus',
```

Comment:
Changes here can be ignored, just removed some commented code and ran a formatter on it